openslides/agenda/static/js/agenda/site.js

```diff
@@ -134,8 +191,19 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
         // delete selected item
         $scope.delete = function () {
             angular.forEach($scope.items, function (item) {
-                if (item.selected)
-                    Agenda.destroy(item.id);
+                if (item.selected) {
+                    switch (item.content_object.collection) {
+                        case 'core/customslide':
+                            Customslide.destroy(item.content_object.id);
+                            break;
+                        case 'motions/motion':
+                            Motion.destroy(item.content_object.id);
+                            break;
+                        case 'assignments/assignment':
+                            Assignment.destroy(item.content_object.id);
+                            break;
+                    }
+                }
```

Comment:
I changed it back to `Agenda.destroy(item.id)`.
But now, this code does not allowed me to delete an agenda item (Customslide, Motion, Assignment)! I get a `403 Forbidden` for the DELETE request (e.g. `DELETE http://localhost:8000/rest/agenda/item/10/`).